build-spark-push.sh

```diff
@@ -0,0 +1,6 @@
+docker build -t openshift-spark  .
+export SPARK_IMAGE=docker.io/analyticsci/openshift-spark:v2.1.0-metrics-prometheus
```

Comment:
Since this has PR has been waiting for awhile, spark has moved to 2.2.0. This tag should probably change to 2.2.0

Reply:
k